docs/index.md

```diff
@@ -57,4 +57,14 @@ snap install freecad --channel <experimental-build-name>
 When you're done testing the build, to return to the original build you were running just swap out the build name and refresh; for example: `stable` or `edge` 
 ```shell
 snap refresh freecad --channel edge
-```
\ No newline at end of file
+```
+
+## Running multiple snaps AKA 'Parallel Installs' (parallel but separate)
+
+By default, snap packages that have several 'channels' will share configs between them. For testing purposes sometimes this isn't wanted, the solution per the snapcraft docs is using the [parellel install](https://snapcraft.io/docs/parallel-installs) feature. 
+
+> Parallel installs enable you to run multiple instances of the same snap on the same system. Each instance is completely isolated from all other instances, including its name, configuration, interface connections, data locations, services, applications and aliases.
+
+Note: at the time of writing this the parallel install feature is still considered experimental
+
+In the context of FreeCAD, this feature would be useful for example if there are experimental changes (in an experimental build) to the FreeCAD config files that could corrupt the snap stable or snap edge channel FreeCAD config files.   
```

Comment:
```suggestion
In the context of FreeCAD, this feature would be useful for example if there are experimental changes (in an experimental build) to the FreeCAD config files that could corrupt the snap stable or snap edge channel FreeCAD config files.   

Example: Install edge snap in parallel to the stable snap

```shell
# Enable experimental parallel instances feature
$ sudo snap set system experimental.parallel-instances=true
# Install freecad snap from channel edge as 'freecad_edge'
$ sudo snap install freecad_edge --channel=edge
# run FreeCAD from this parallel install
$ freecad_edge.freecad
```
```
```